conans/client/tools/oss.py

```diff
@@ -221,10 +224,55 @@ def get_solaris_version_name(version):
         elif version.minor() == "5.11":
             return "Solaris 11"
 
+    @staticmethod
+    def bash_path():
+        if "CONAN_BASH_PATH" not in os.environ:
+            bash_path = None
+            for ext in [".bat", ".exe", ".sh", ""]:
```

Comment:
So, will keep checking? Why not break?

Reply:
I think I tried to create kind of priority doing that. I'll check.